src/frontend/src/optimizer/plan_node/generic/agg.rs

```diff
@@ -361,6 +361,66 @@ impl<PlanRef: stream::StreamPlanRef> Agg<PlanRef> {
         internal_table_catalog_builder.build(tb_dist)
     }
 
+    /// Infer dedup tables for distinct agg calls, partitioned by distinct columns.
+    /// Since distinct agg calls only dedup on the first input column, the key of the result map is
```

Comment:
What is "first input column" referring to?

Reply:
Oh I meant first argument column of the call.